quiz/utils.py

```diff
@@ -0,0 +1,34 @@
+
+from django.db.models.fields import SlugField
+from django.shortcuts import get_object_or_404
+from organization.models import InternalGroup
+from users.models import User
+from quiz.models import Participant, Quiz
+
+
+def user_quiz_pool_helper(quiz, internal_group):
```

Comment:
Would prefer a bit more descriptive function name here as well, maybe something like `generate_fake_user_pool` or `bootstrap_mock_users` and a little docstring briefly explaining what is happening